docs/Release_Board-Maintainers.md

```diff
@@ -82,6 +82,7 @@ The authoritative list of `board.conf` can be found [here](https://github.com/ar
 | orangepiwin          |                |          |
 | orangepizero         | igorpecovnik   |          |
 | orangepizero2        | krachlatte     |          |
+|"                     | qiurui         |          |
```

Comment:
Why its " here?

Reply:
Meaning maintaining the same board as above